Makefile

```diff
@@ -23,8 +23,25 @@ DOCKER_RUN := mkdir -p ../.go-pkg-cache bin $(GOMOD_CACHE) && \
 MAKE_DIRS=$(shell ls -d */)
 
 generate:
-	make -C api gen-files 
-	make -C libcalico-go gen-files
+	$(MAKE) -C api gen-files
```

Comment:
Actually i'm about to add a clean make target and get rid of the rm app-policy/proto line below.  That should not be there.